nad

```diff
@@ -920,7 +928,14 @@ function setup_reverse() {
       } else {
         console.log("configuration error for reverse: no check found (hostname mismatch?)");
       }
-  });
+  }
+
+  circapi.setup(auth_token, 'nad', api_options);
+  if (check_bundle_id == null) {
+      circapi.get('/check_bundle?f_type=json:nad&f_target=' + hostname, null, circapi_cb);
+  else {
+      circapi.get('/check_bundle/' + check_bundle_id, null, circapi_cb);
```

Comment:
I went over this with mgm.  He said check_bundle is what we want here as the check object doesn't contain the reverse urls.